### PR TITLE
Try to ensure manage.py is launched using py3.

### PR DESCRIPTION
**Pull request by FeralBytes**
Will resolve issue #53 .

### CODE REVIEW

File: testserver/manage.py [hunk 1]
```diff
@@ -3,6 +3,15 @@
 import sys
 
 if __name__ == "__main__":
+    
+    if 'python3' not in sys.executable:  # might be better to check version also.
```
Comment (funkybob) [inline testserver/manage.py]: Why not:

    if sys.version_info[0] != 3:

File: testserver/manage.py [hunk 2]
```diff
@@ -3,6 +3,15 @@
...
+    # if sys.hexversion < 0x03000000:
+        import distutils.spawn
+        python3_path = distutils.spawn.find_executable('python3')
+        if python3_path is not None:
+            os.execvp(sys.executable + '3', [sys.executable + '3'] + sys.argv)
+            #TODO what to do if it is None?
```
Comment (freakboy3742) [inline testserver/manage.py]: What to do:
```
print('Batavia requires Python 3', file=sys.stderr)
sys.exit(1)
```

Comment (FeralBytes) [inline testserver/manage.py]: `print('Batavia requires Python 3', file=sys.stderr)`
Is not compatible with Python 2. So I changed it too:
`sys.stderr.write('Batavia requires Python 3' + os.linesep)`

File: testserver/manage.py (hunk 1 above)
Comment (FeralBytes) [inline testserver/manage.py]: Good point; I had not arrived at that solution; but I like it better. Implemented.

File: testserver/manage.py [hunk 3]
```diff
@@ -3,6 +3,16 @@
...
 
 if __name__ == "__main__":
+    
+    if sys.version_info[0] != 3:
+        import distutils.spawn
+        python3_path = distutils.spawn.find_executable('python3')
+        if python3_path is not None:
+            os.execvp(sys.executable + '3', [sys.executable + '3'] + sys.argv)
+        else:
+            sys.stderr.write('Batavia requires Python 3' + os.linesep)
```
Comment (freakboy3742) [inline testserver/manage.py]: The spawn check isn't needed - having python2 start python3 is just going to lead to confusion. If sys.version_info doesn't return the right result, it's better to throw an error and let the user deal with that.

Comment (FeralBytes) [inline testserver/manage.py]: OK. I disagree; but it is your project.

**Comment by freakboy3742** [comment]
Thanks for the contribution! Hopefully nobody else will get bitten by accidentally using Python2.

**Pull request merged**